src/seedu/addressbook/Main.java

```diff
@@ -9,6 +9,7 @@
 import seedu.addressbook.storage.StorageFile;
 import seedu.addressbook.ui.TextUi;
 
+import java.io.FileNotFoundException;
```

Comment:
So many unrelated commits. So it is very hard to review your code. Avoid this next time please.